java/src/main/java/io/cucumber/createmeta/CodeGenerator.java

```diff
@@ -0,0 +1,67 @@
+package io.cucumber.createmeta;
+
+import io.cucumber.messages.internal.com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import io.cucumber.messages.internal.com.fasterxml.jackson.databind.DeserializationFeature;
+import io.cucumber.messages.internal.com.fasterxml.jackson.databind.ObjectMapper;
+import io.cucumber.messages.internal.com.fasterxml.jackson.databind.type.MapType;
+
+import java.io.File;
+import java.io.IOException;
+import java.text.MessageFormat;
+import java.util.Map;
+import java.util.SortedMap;
+import java.util.stream.Collectors;
+
+// https://newbedev.com/generating-sources-by-running-a-project-s-java-class-in-maven
+public class CodeGenerator {
```

Comment:
```suggestion
public final class CodeGenerator {
```

Reply:
The entire CodeGenerator class was an experiment that I forgot to delete. It's deleted now.